src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/HomePage.java

```diff
@@ -2,6 +2,7 @@
 
 import com.wikia.webdriver.common.core.configuration.Configuration;
 
+import com.google.common.base.Function;
 import com.google.common.base.Predicate;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Predicate'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)